pettingzoo/classic/chess/chess_utils.py

```diff
@@ -321,17 +324,18 @@ def get_observation(orig_board, player):
       }
     """
     # from 0-63
-    square = board.ep_square
+    # Adjust the row number for the white pawn to the 8th if the en passant flag is set, and vice versa for black pawns.
+    square = board.ep_square  # (int) where the en passant happened
     if square:
-        ours = square > 32
+        ours = square < 32
```

Comment:
I'm assuming this is because ours was flipped to be the opposite in the line above, but could leave an inline comment saying `less than 32 is a white square` (or whichever color it is, I'm a little mixed up with all these details)

Reply:
I add a comment of `Less than 32 is a white square, otherwise it's a black square` above it.